src/widget/wtrackproperty.cpp

```diff
@@ -42,8 +42,8 @@ void WTrackProperty::slotLoadingTrack(TrackPointer pNewTrack, TrackPointer pOldT
 
 void WTrackProperty::updateLabel(Track* /*unused*/) {
     if (m_pCurrentTrack) {
-        QVariant property = m_pCurrentTrack->property(m_property.toAscii().constData());
-        if (property.isValid() && qVariantCanConvert<QString>(property)) {
+        QVariant property = m_pCurrentTrack->property(m_property.toLatin1().constData());
```

Comment:
I think this should be utf8() since it is used everywhere else when reading characters from cpp files in QT5. 
The property main is defined with Q_PROPERTY in this way.

Reply:
Done